src/helpers/linkHelper.js

```diff
@@ -20,7 +20,12 @@ function ensureProtocol(link) {
     linkWithProtocol = `http://${link}`;
   }
 
-  return linkWithProtocol;
+  return spaceDeletion(linkWithProtocol);
```

Comment:
i suggest `link.replaceAll(' ', '');` when setting `let linkWithProtocol = link` instead of creating a method that is used only once? and i see no reason in removing spaces from `linkWithProtocol` as it is the `link` prefixed with http://

Reply:
Yes. It is unnecessary to create a new method for a feature that is used once. I just created it to fit the structure in the code. I have updated it here: https://github.com/smart-village-solutions/smart-village-app-app/commit/b44a3ca6bb3c3e1d8354bf7f505c88719a9f0cba